From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Steve Muckle <smuckle@google.com>
Date: Thu, 31 Oct 2019 15:38:06 -0700
Subject: ANDROID: gki_defconfig: enable CONFIG_KEYBOARD_GPIO

This driver is very commonly used in Android. Statically compile it into
the GKI.

Bug: 143314501
Change-Id: I12e2dd04bdde2affef1f6be9f33033d87096ec89
Signed-off-by: Steve Muckle <smuckle@google.com>
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 1d20a9275cc8..54d4704f9ea0 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -263,6 +263,7 @@ CONFIG_USB_USBNET=y
 # CONFIG_WLAN_VENDOR_QUANTENNA is not set
 CONFIG_VIRT_WIFI=y
 CONFIG_INPUT_EVDEV=y
+CONFIG_KEYBOARD_GPIO=y
 # CONFIG_INPUT_MOUSE is not set
 CONFIG_INPUT_JOYSTICK=y
 CONFIG_INPUT_MISC=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index 958bff41b3fb..3be71fa9a2c4 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -237,7 +237,7 @@ CONFIG_USB_USBNET=y
 # CONFIG_WLAN_VENDOR_QUANTENNA is not set
 CONFIG_VIRT_WIFI=m
 CONFIG_INPUT_EVDEV=y
-# CONFIG_INPUT_KEYBOARD is not set
+CONFIG_KEYBOARD_GPIO=y
 # CONFIG_INPUT_MOUSE is not set
 CONFIG_INPUT_JOYSTICK=y
 CONFIG_INPUT_MISC=y
